app/utils/web3_utils.py

```diff
@@ -27,11 +13,10 @@ def get_web3_from_chain_id(chain_id):
         raise NotImplementedError
     else:
         endpoint = xy_config.WEB3_RPC_ENDPOINTS['SWAP'][chain_id]
-
     w3 = Web3(
         Web3.HTTPProvider(endpoint, request_kwargs={"timeout": 30}),
     )
-    if chain_id in POA_CHAINS:
+    if chain_id in xy_config.IS_POA_MIDDLEWARE_REQUIRED:
```

Comment:
阿抱歉應該是要用 `IS_POA_MIDDLEWARE_REQUIRED[chain_id]`，我之前 review 寫錯了。
```suggestion
    if xy_config.IS_POA_MIDDLEWARE_REQUIRED[chain_id]:
```

Reply:
啊我也是無腦貼沒注意到